conf/examples/acm_hub_unreleased_image.yaml

```diff
@@ -1,3 +1,3 @@
 ---
 ENV_DATA:
-  acm_unreleased_image: "2.5.2-DOWNSTREAM-2022-07-29-15-56-49"
+  acm_unreleased_image: "2.7.0-DOWNSTREAM-2022-11-17-03-53-08"
```

Comment:
@prsurve I saw your comment that you have updated the image to the latest downstream tag but here it is pointing to a specific image

Reply:
Yes we have to add the downstream tag as there is no latest tags present in the ACM repository